upload-api/tables/index.js

```diff
@@ -26,3 +26,55 @@ export const uploadTableProps = {
   // space + root must be unique to satisfy index constraint
   primaryIndex: { partitionKey: 'space', sortKey: 'root' },
 }
+
+/** @type TableProps */
+export const delegationTableProps = {
+  fields: {
+    link: 'string',        // `baf...x`
+    audience: 'string',   // `did:web:service`
+    issuer: 'string',     // `did:key:agent`
+    expiration: 'string', // `9256939505` (unix timestamp)
+    insertedAt: 'string', // `2022-12-24T...`
+    updatedAt: 'string',  // `2022-12-24T...`
```

Comment:
I think we need `cause` field for the invocation CID here as well, especially if we'll be looking things up from the UCAN log.

Reply:
yep! that made it in in the PR that moved to using delegations from the invocations they came in